src/OpenApi/src/Microsoft.AspNetCore.OpenApi.csproj

```diff
@@ -4,14 +4,17 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;openapi</PackageTags>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Why?

Reply:
This can go away. It's an artifact from when I was experimented with different buffered writers. 😬

Reply:
JK -- this needs to stay because of the Utf8BufferTextWriter we reused from SignalR. I'll post an inline comment about it.